astroid/brain/brain_pytest.py

```diff
@@ -18,19 +18,62 @@ def pytest_transform():
     import _pytest.recwarn
     import _pytest.runner
     import _pytest.python
+    import _pytest.skipping
+    import _pytest.assertion
 except ImportError:
     pass
 else:
     deprecated_call = _pytest.recwarn.deprecated_call
+    warns = _pytest.recwarn.warns
+    xfail = _pytest.skipping.xfail
     exit = _pytest.runner.exit
     fail = _pytest.runner.fail
-    fixture = _pytest.python.fixture
     importorskip = _pytest.runner.importorskip
     mark = _pytest.mark.MarkGenerator()
     raises = _pytest.python.raises
     skip = _pytest.runner.skip
-    yield_fixture = _pytest.python.yield_fixture
 
+    # New in pytest 3.0
+    try:
+        approx = _pytest.python.approx
+        register_assert_rewrite = _pytest.assertion.register_assert_rewrite
+    except AttributeError:
+        pass
+
+
+# Moved in pytest 3.0
+
+try:
+    import _pytest.freeze_support
+    freeze_includes = _pytest.freeze_support.freeze_includes
```

Comment:
I decided to do the assignments here instead of in an `else:` as there's no way this second line could raise an `ImportError`, and this makes the nested `ImportError`s much more readable IMHO.